### PR TITLE
Update README.md - Enhance Hubot commands and remove Campfire support

### PR DESCRIPTION
**Pull request by gargola**
Starting a PR to update README.md. Here I enhance some Hubot commands by adding "templated" examples.

## Checklist

Feel free to adjust this checklist as necessary:

- [x] To get review.
- [ ] Merge and :ice_cream:
- [ ] Please remember to delete your branch after merging :heart:

### CODE REVIEW

**Comment by mattr-** [comment]
Submitted against the wrong repository.

**Comment by mattr-** [comment]
Campfire I'm willing to remove support for. HipChat, I'm not. Could you add this section back please?

**Comment by gargola** [comment]
Got it. Will do it right now.

**Comment by gargola** [comment]
@mattr- , done!

File: README.md [hunk 1]
```diff
@@ -214,15 +208,7 @@ Required settings:
   rather than email
 * `JANKY_CHAT_DEFAULT_ROOM` should be the name of the room instead of the
   XMPP format, for example: `Engineers` instead of xxxx_xxxxxx.
-
```
Comment (mattr-) [inline README.md]: Sorry, I wasn't clear enough before, since this installation section below is also for HipChat, let's not remove it in this PR.

**Comment by mattr-** [comment]
Awesome! Thanks!

**Pull request merged**